contracts/external/compound/CTokenInterface.sol

```diff
@@ -1,12 +1,14 @@
 pragma solidity 0.6.4;
 
-interface CTokenInterface {
-    function totalSupply() external view returns (uint256);
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
+
+interface CTokenInterface is IERC20 {
+    function totalSupply() external override view returns (uint256);
```

Comment:
✂️

Reply:
IERC20 also has a totalSupply() so this has to be "override"